src/test/java/seedu/address/logic/parser/SharkieParserTest.java

```diff
@@ -0,0 +1,113 @@
+                instanceof PeopleClearCommand);
+        assertTrue(parser.parseCommand(PEOPLE_COMMAND_TYPE + " " + PeopleClearCommand.COMMAND_WORD + " 3")
+                instanceof PeopleClearCommand);
+    }
+
+    @Test
+    public void parseCommand_delete() throws Exception {
+        PeopleDeleteCommand command = (PeopleDeleteCommand) parser.parseCommand(PEOPLE_COMMAND_TYPE + " "
+                + PeopleDeleteCommand.COMMAND_WORD + " " + INDEX_FIRST_PERSON.getOneBased());
+        assertEquals(new PeopleDeleteCommand(INDEX_FIRST_PERSON), command);
+    }
+
+    @Test
+    public void parseCommand_edit() throws Exception {
+        Person person = new PersonBuilder().build();
+        EditPersonDescriptor descriptor = new EditPersonDescriptorBuilder(person).build();
+        PeopleEditCommand command = (PeopleEditCommand) parser.parseCommand(PEOPLE_COMMAND_TYPE + " "
+                + PeopleEditCommand.COMMAND_WORD + " " + INDEX_FIRST_PERSON.getOneBased() + " "
+                + PersonUtil.getEditPersonDescriptorDetails(descriptor));
+        assertEquals(new PeopleEditCommand(INDEX_FIRST_PERSON, descriptor), command);
+    }
+
+    @Test
+    public void parseCommand_exit() throws Exception {
+        assertTrue(parser.parseCommand(SHARKIE_COMMAND_TYPE + " " + ExitCommand.COMMAND_WORD)
+                instanceof ExitCommand);
+        assertTrue(parser.parseCommand(SHARKIE_COMMAND_TYPE + " " + ExitCommand.COMMAND_WORD + " 3")
+                instanceof ExitCommand);
+    }
+
+    @Test
+    public void parseCommand_find() throws Exception {
+        List<String> keywords = Arrays.asList("foo", "bar", "baz");
+        PeopleFindCommand command = (PeopleFindCommand) parser.parseCommand(
+                PEOPLE_COMMAND_TYPE + " " + PeopleFindCommand.COMMAND_WORD + " "
+                        + keywords.stream().collect(Collectors.joining(" ")));
+        assertEquals(new PeopleFindCommand(new NameContainsKeywordsPredicate(keywords)), command);
+    }
+
+    @Test
+    public void parseCommand_help() throws Exception {
+        assertTrue(parser.parseCommand(SHARKIE_COMMAND_TYPE + " " + HelpCommand.COMMAND_WORD)
+                instanceof HelpCommand);
+        assertTrue(parser.parseCommand(SHARKIE_COMMAND_TYPE + " " + HelpCommand.COMMAND_WORD + " 3")
+                instanceof HelpCommand);
+    }
+
+    @Test
+    public void parseCommand_list() throws Exception {
+        assertTrue(parser.parseCommand(PEOPLE_COMMAND_TYPE + " " + PeopleListCommand.COMMAND_WORD)
+                instanceof PeopleListCommand);
+        assertTrue(parser.parseCommand(PEOPLE_COMMAND_TYPE + " " + PeopleListCommand.COMMAND_WORD + " 3")
+                instanceof PeopleListCommand);
+    }
+
+    @Test
+    public void parseCommand_unrecognisedInput_throwsParseException() {
+        assertThrows(ParseException.class, String.format(MESSAGE_INVALID_COMMAND_FORMAT, HelpCommand.MESSAGE_USAGE), ()
+            -> parser.parseCommand(""));
+    }
+
```

Comment:
Maybe could add a test for commands that should have the prefix in front (e.g. people edit without the people)